common/changes/office-ui-fabric-react/focus2_2018-03-09-17-45.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Removed focusability on buttons for split buttons and spin buttons",
+      "type": "patch"
```

Comment:
This isn't a patch. This is a minor change. Any time you're adding a type to a public interface, you're not forwards compatible.